src/site/resources/js/checkstyle.js

```diff
@@ -17,6 +17,34 @@ window.addEventListener("load", function () {
     });
 });
 
+document.addEventListener("DOMContentLoaded", () => {
+    const url = new URL(window.location.href);
+    if (!url.pathname.endsWith("/releasenotes.html")
+            || !url.hash.startsWith("#Release_")
+            || document.getElementById(url.hash.replace("#", ""))) {
+        return;
+    }
+
+    const version = url.hash.split('_')[1];
```

Comment:
This `[1]` access is safe, because the `if`-statement above ensures that `url.hash` contains `Release_`.